example/main.dart

```diff
@@ -33,6 +33,12 @@ class MyApp extends StatelessWidget {
                     child: const Text('Check Update'),
                     onPressed: () => siren.promptUpdate(context),
                   )
+                  Text(
+                    'Local Version: ${siren.localVersion}',
```

Comment:
This example isn't working so well, what do you think about rewriting using FutureBuilder or other ways to work with Future on Flutter?

![Simulator Screen Shot - iPhone 13 - 2021-10-30 at 22 07 59](https://user-images.githubusercontent.com/3134422/139562930-f1b1c435-14cf-41f8-889a-bb1e74e3d272.png)

Reply:
Oups my bad :/ I'll be on that 😄